src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArr = sourceString.split(';');
+  const newArr = sourceArr.map((el) => el.trim());
```

Comment:
Avoid variable namings like `array`, `arr`, `newArr`, as they decrease code readability. Instead, use descriptive names that reflect the content or purpose of the array. For example, `styleDeclarations` would be more descriptive.